app/3.0/docs/tools/polymer-cli-commands.md

```diff
@@ -232,11 +267,47 @@ App project demo:
 
 ### Server options {#server-options}
 
-This section shows examples of using various `polymer serve` options.
+This section describes command line options available for the Polymer CLI development server (`polymer serve`).
+
+### --npm
+
+Sets npm mode. Dependencies are installed from npm, the component directory is set to `node_modules` and the package name is read from `package.json`.
+
+```
+polymer serve --npm --module-resolution="node"
+```
+
+### --module-resolution
 
-Serve from port 3000:
+Specifies how to resolve module specifiers in import and export statements when rewriting them to be web-compatible.
```

Comment:
This could use a link to something that describes that web-compatible imports are path-only, though I don't have a great reference in mind right now.

Reply:
maybe http://2ality.com/2017/05/es-module-specifiers.html#how-do-module-specifiers-change-with-esm, wdyt?